sgkit/stats/ld.py

```diff
@@ -61,3 +70,377 @@ def rogers_huff_r_between(gn0: ArrayLike, gn1: ArrayLike) -> float:  # pragma: n
 @njit(nogil=True, fastmath=True, cache=True)  # type: ignore
 def rogers_huff_r2_between(gn0: ArrayLike, gn1: ArrayLike) -> float:  # pragma: no cover
     return rogers_huff_r_between(gn0, gn1) ** 2  # type: ignore
+
+
+def ld_matrix(
+    ds: Dataset,
+    *,
+    dosage: Hashable = variables.dosage,
+    threshold: Optional[float] = None,
+    ld_score: Optional[Hashable] = None,
```

Comment:
🤔  Not a strong opinion but `ld_score` is maybe a little too polysemous for what it represents here?  Perhaps `variant_score` or just `score` instead?

Reply:
Yes `LD score` is a term of art with a precise definition (cf [slide 15](http://ibg.colorado.edu/cdrom2016/nealeB/LDScore/LDScore_2016_Boulder.pdf)) so we should use a different name here.

Reply:
Good point. Renamed to `variant_score` (including the name of the variable).